pkgs/top-level/python-packages.nix

```
@@ -1590,7 +1590,9 @@ in {
 
   cryptacular = callPackage ../development/python-modules/cryptacular { };
 
-  cryptography = callPackage ../development/python-modules/cryptography { };
+  cryptography = callPackage ../development/python-modules/cryptography {
+    inherit (pkgs.darwin) libiconv;
```

```suggestion
    inherit (darwin) libiconv;
```

Nitpick: `pkgs.` shouldn't be required here (or is it relevant for something?).
You can also `inherit (darwin.apple_sdk.frameworks) Security` for consistency, if you want / don't mind.

Anyway, LGTM, thanks!
(I cannot test/verify Darwin changes but CI should.)

---

I would say it is. If someone adds a `darwin` python package, then callPackage will use that package.

---

Without `pkgs` I get:

```
  1593|   cryptography = callPackage ../development/python-modules/cryptography {
  1594|     inherit (darwin) libiconv;
      |              ^
  1595|   };

undefined variable 'darwin'
```

---

Oops, my bad, sorry :o My mind was in `all-packages.nix` but this is obviously for `python-packages.nix`.